README.md

```diff
@@ -1,21 +1,77 @@
 # golf-course
 A list of common Solidity optimization tips and myths.
 
-## Tips
+# Tips
 
-### Right Shift Instead of Dividing By 2
+- - - -
+### 1.  When dividing by two, use `>> 1` instead of `/ 2` ###
+
+```solidity
+/// 🤦 Unoptimized (gas: 1401)
```

Comment:
what do you think of showing the gas from the test here?

Reply:
love it, tho may get hard to maintain after sol upgrades etc...